erpnext/setup/doctype/item_group/item_group.py

```diff
@@ -37,30 +37,34 @@ def validate(self):
 	def on_update(self):
 		NestedSet.on_update(self)
 		invalidate_cache_for(self)
+		self.make_route()
```

Comment:
why called make_route from on_update, it's already called from validate

Reply:
Sorry, didn't notice that. Already remove `make_route` in `on_update`